pkg/setting/setting_unified_alerting.go

```diff
@@ -95,47 +95,50 @@ func (u *UnifiedAlertingSettings) IsEnabled() bool {
 	return u.Enabled == nil || *u.Enabled
 }
 
+// readUnifiedAlertingEnabledSettings reads the settings for unified alerting.
+// It returns a non-nil bool and a nil error when unified alerting is enabled either
+// because it has been enabled in the settings or by default. It returns nil and
+// a non-nil error both unified alerting and legacy alerting are enabled at the same time.
 func (cfg *Cfg) readUnifiedAlertingEnabledSetting(section *ini.Section) (*bool, error) {
-	enabled, err := section.Key("enabled").Bool()
-	// the unified alerting is not enabled by default. First, check the feature flag
+	// At present an invalid value is considered the same as no value. This means that a
+	// spelling mistake in the string "false" could enable unified alerting rather
+	// than disable it. This issue can be found here
+	unifiedAlerting, err := section.Key("enabled").Bool()
```

Comment:
Should we take the opportunity to fix this? 

I think:

```
section.Key("enabled").MustBool(false)
```

Should be a safe bet to disable UA if there's ambiguity in the value?

Reply:
I spoke to Armand and we can fix this in a follow up due to the release going out tomorrow. It might require more careful changes and tests.